### PR TITLE
[Term Entry] PHP Math Functions exp() #2269

### PR DESCRIPTION
**Pull request by MMMMMMMZ**
### Description

addresses issue #2269

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/exp/exp.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Exp()'
+Description: 'Returns E raised to the power of x (E^x^).'
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/exp/exp.md:3]:
```suggestion
Description: 'Returns e raised to the power of x.'
```

File: content/php/concepts/math-functions/terms/exp/exp.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/exp/exp.md:4]:
```suggestion
```
Remove blank line

File: content/php/concepts/math-functions/terms/exp/exp.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/exp/exp.md:9]:
```suggestion
```
Remove blank line

File: content/php/concepts/math-functions/terms/exp/exp.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Tags:
+  - 'Strings'
+  - 'Functions'
+
+CatalogContent:
+  - 'learn-php'
+
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/exp/exp.md:16]:
```suggestion
```
Remove blank line

File: content/php/concepts/math-functions/terms/exp/exp.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  - 'paths/computer-science'
+---
+
+The `exp()` function Returns E raised to the power of x (E^x^).
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/exp/exp.md:20]:
```suggestion
The **`exp()`** function Returns _e_ raised to the power of x.
```

File: content/php/concepts/math-functions/terms/exp/exp.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+## Syntax
+
+```pseudo
+
+exp($x)
+
+```
+
+- `$x`: specifies the exponent that E will be raised to.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/exp/exp.md:30]:
```suggestion
- `$x`: specifies the exponent that _e_ will be raised to.
```

File: content/php/concepts/math-functions/terms/exp/exp.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+  - 'paths/computer-science'
+---
+
+The `exp()` function Returns E raised to the power of x (E^x^).
+
+## Syntax
+
+```pseudo
+
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/exp/exp.md:25]:
```suggestion
```
Remove blank line

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@MMMMMMMZ I did a first review and left a number of comments. Many of them just removing excess blank lines.

**Comment by MMMMMMMZ** [comment]
I added/removed everything you told me to

**Review by SSwiniarski — APPROVED** [review]
@MMMMMMMZ I made one minor edit. Now it looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@MMMMMMMZ @SSwiniarski Looks good to merge.

**Comment by SSwiniarski** [comment]
@MMMMMMMZ Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/math-functions/exp

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**